tf_from_ar_marker_to_sandwich:
  # marker_size : 3cm
  markers:
    front:
      translation:
        # x: -(marker_size / 2) * cos()
        # z: sandwich_height - (marker_size / 2) * sin()
        x: -0.01003447097
        y: 0.000000
        z: 0.0888505871
      rotation:
        # roll: Atan(sandwich_height / sandwich_depth)
        roll: 0.837981225
        pitch: 0.000000
        yaw: -1.570796
    back:
      translation:
        x: 0.000000
        y: 0.000000
        # z: sandwich_height - (marker_size / 2)
        z: 0.085
      rotation:
        roll: 1.570796
        pitch: 0.000000
        yaw: 1.570796
    bottom:
      translation:
        # x: -sandwich_depth + (marker_size / 2)
        x: -0.075
        y: 0.000000
        z: 0.000000
      rotation:
        roll: 0.000000
        pitch: 3.141593
        yaw: 1.570796
    right:
      translation:
        # x: -(marker_size / 2)
        # y: -(sandwich_width / 2)
        # z: (marker_size / 2)
        x: -0.015
        y: -0.04000
        # y: -0.035000
        z: 0.015
      rotation:
        roll: 1.570796
        pitch: 0.00000
        yaw: 0.0000000
    left:
      # x: -(marker_size / 2)
      # y: (sandwich_width / 2)
      # z: (marker_size / 2)
      translation:
        x: -0.015
        y: 0.04000
        # y: 0.035000
        z: 0.015
      rotation:
        roll: 1.570796
        pitch: 0.00000
        yaw: 3.141593
  # 5cm
  # markers:
  #   front:
  #     translation:
  #       x: -0.013379
  #       y: 0.000000
  #       z: 0.114866
  #     rotation:
  #       roll: 0.837981
  #       pitch: 0.000000
  #       yaw: -1.570796
  #   back:
  #     translation:
  #       x: 0.000000
  #       y: 0.000000
  #       z: 0.080000
  #     rotation:
  #       roll: 1.570796
  #       pitch: 0.000000
  #       yaw: 1.570796
  #   bottom:
  #     translation:
  #       x: -0.070000
  #       y: 0.000000
  #       z: 0.000000
  #     rotation:
  #       roll: 0.000000
  #       pitch: 3.141593
  #       yaw: 1.570796
  #   right:
  #     translation:
  #       x: -0.025
  #       y: -0.035000
  #       z: 0.025
  #     rotation:
  #       roll: 1.570796
  #       pitch: 0.00000
  #       yaw: 0.0000000
  #   left:
  #     translation:
  #       x: -0.025
  #       y: 0.035000
  #       z: 0.025
  #     rotation:
  #       roll: 1.570796
  #       pitch: 0.00000
  #       yaw: 3.141593
